_posts/en/newsletters/2021-01-06-newsletter.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Bitcoin Optech Newsletter #130'
+permalink: /en/newsletters/2021/01/06/
+name: 2021-01-06-newsletter
+slug: 2021-01-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed update to BIP322 generic
+message signing and links to a specification for LN trampoline routing.
+Also included are our regular sections with announcements of releases,
+release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Proposed updates to generic signmessage:** Andrew Poelstra
+  [posted][poelstra post] a partial rewrite of the [BIP322][]
+  generic message signing proposal.  The rewrite mainly clarifies the
+  signing and verification procedures, but it now also allows wallets
+  that don't implement a complete set of checks to return an
+  "inconclusive" state for signatures that use scripts they don't
+  understand.  For example, a developer who wants to quickly implement
+  generic signmessage can only implement signature verification for
```

Comment:
maybe s/can/might|might choose to/ 

(to clarify that its a choice by the developer)